staging/src/k8s.io/client-go/util/certificate/csr/csr.go

```diff
@@ -115,8 +115,13 @@ func WaitForCertificate(ctx context.Context, client certificatesclient.Certifica
 				if c.Type == certificates.CertificateDenied {
 					return false, fmt.Errorf("certificate signing request is not approved, reason: %v, message: %v", c.Reason, c.Message)
 				}
-				if c.Type == certificates.CertificateApproved && csr.Status.Certificate != nil {
-					return true, nil
+				if c.Type == certificates.CertificateApproved {
+					if csr.Status.Certificate != nil {
+						klog.V(2).Infof("certificate signing request is issued")
```

Comment:
If we want this for debugging we may want to add the name to these debug logs. (csr.Name)

V2 seems high especially for the everything good case. Do we know how often this will be called?

Even for the unissued case, its not clear to me we want print out the message repeatedly.

Reply:
These two lines will be printed at most once, given that in our current implementation, the csr should be either denied or approved at most once. By "each attempt" I mean if the csr is denied, kubelet will fail and restart, then the messages will repeat. I have run e2e test with modified controller manager to verify the said behavior.

I will dump `csr` to see what should be included. Will update when finished.

Reply:
- `csr.Name` add
- V(3) is already as low as network traffic, I think we should keep it V(2)
- verified no spamming during bootstrap test nodes (in presubmit tests)